frontend/src/scenes/surveys/Surveys.stories.tsx

```diff
@@ -363,7 +363,7 @@ export const NewSurveyWithHTMLQuestionDescription: StoryFn = () => {
 NewSurveyWithHTMLQuestionDescription.parameters = {
     testOptions: {
         waitForSelector:
-            '#survey > div.flex.flex-row.gap-4 > div.max-w-80.mx-4.flex.flex-col.items-center.h-full.w-full.sticky.top-0.pt-16 > div > div:nth-child(1) > form > div > div > div:nth-child(2) > div.description > strong',
+            '#survey > div.flex.flex-row.gap-4 > div.max-w-80.mx-4.flex.flex-col.items-center.h-full.w-full.sticky.top-0.pt-16 > div > div:nth-child(1) > form > div > div:nth-child(2) > div:nth-child(1) > div.description > strong',
```

Comment:
This seems a bit brittle. Does it make sense to add an id or data-attr for tests?

Reply:
yeah that's exactly what I'm doing – requires a PR to posthog-js though since we do all the rendering of the DOM in that library and then import it into posthog for the preview.  This fix just unblocks the build.